docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -57,6 +57,26 @@ Such arguments could include passwords or other sensitive information.
 You should also make sure no stack trace is ever visible to end users of production sites,
 though visible arguments are unsafe even if the stack traces only show up in log files.
 
+### Disable error output from PHP
+
+Symfony in production mode prevents exception messages from being visible to end users.
+However, if Symfony fails to boot properly, such exceptions may end up being visible, including stack traces.
+This can be prevented by [disabling error message output in PHP](https://www.php.net/manual/en/language.errors.basics.php).
+These `php.ini` configuration values should be used on production sites.
+When using '[[= product_name_cloud =]]' / Platform.sh the same settings can be configured in '[[= product_name_base =]]' DXP's `.platform.app.yaml` file.
+
+    ```ini
+    display_errors          = Off
+    display_startup_errors  = Off
+    ```
```

Comment:
Please remove the indent, it's not correctly formatted right now (https://ez-systems-developer-documentation--2423.com.readthedocs.build/en/2423/infrastructure_and_maintenance/security/security_checklist/#enable-zendexception_ignore_args-in-php-74-and-newer)

(I tried to create a suggestion from GitHub, but it doesn't handle nested backticks well 😅 )

Reply:
````suggestion
```ini
display_errors          = Off
display_startup_errors  = Off
```
````